wherehows-dao/src/main/java/wherehows/dao/CFGApplicationDao.java

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Can i leave the file here? I created it when refactor MetadataChangeProcessor, later on we discussed leave the processor empty and do refactor work in the future.
I believe this will be needed in the future.